db/migrate/20240321203236_add_sent_for_curation_to_work_version.rb

```diff
@@ -0,0 +1,5 @@
+class AddSentForCurationToWorkVersion < ActiveRecord::Migration[6.1]
+  def change
+    add_column :work_versions, :sent_for_curation, :datetime
```

Comment:
I hate to nitpick, but since this is a datetime we should name it something that ends with `_at`.  So this should be `sent_for_curation_at`.